wizeline/wize-q-remix
# feat: Migrate NPS UI and add Create and Delete Controllers

#### What does this PR do?
- Add the controller to create and delete the nps for the answer.
- Migrate the components related with the nps.

#### How should this be manually tested?
1. Login 
2. Go to for one question with an answer (If you are an administrator, the answer should not belong to you )
3. Select one option for the question, ** Do you consider the answer solves the question?**  and click on `Send`
4. To verify that you can delete your selection, click on `Undo`

1. To test the controllers, run the command `npm run test:integration`

#### What are the relevant tickets?
Closes #63 

#### Screenshots
[App]

https://user-images.githubusercontent.com/97203617/200753094-70ab27df-32cf-4d01-9d42-fafd2af85c0c.mov


[test]
<img width="1044" alt="Screen Shot 2022-11-09 at 0 09 10" src="https://user-images.githubusercontent.com/97203617/200752819-7b6770b7-21ef-431c-81fb-f453826ef7e1.png">
